media/js/firefox/whatsnew/whatsnew-137-uitour.es6.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+function init() {
+    Mozilla.UITour.ping(() => {
+        // main CTA should open about:preferences at "Browser Layout" section
+        const button = document.querySelector('.wnp-main-cta .mzp-c-button');
+
+        button.addEventListener(
+            'click',
+            (e) => {
+                e.preventDefault();
+
+                Mozilla.UITour.openPreferences('browser-layout');
```

Comment:
I haven't been able to verify that this works since UITour is no longer working for me locally using Nightly. Could you perhaps push this to demo so I can test on a secure host?

Reply:
@alexgibson sorry I missed this earlier - it is available on demo at https://www-demo6.allizom.org/en-US/firefox/137.0/whatsnew/

Reply:
Works as expected for me on demo 👍